rotkehlchen/api/rest.py

```diff
@@ -4719,25 +4719,30 @@ def export_history_events(
 
         serialized_history_events = []
         headers: dict[str, None] = {}
+        prices_from_db = GlobalDBHandler.get_historical_prices(
+            query_data=(query_data := [(event.asset, currency, ts_ms_to_sec(event.timestamp)) for event in history_events]),  # type: ignore[arg-type]  # currency is a subclass of Asset  # noqa: E501
+            max_seconds_distance=HOUR_IN_SECONDS,
+        )
+        missing_prices: list[tuple[Asset, Timestamp]] = []
+        cached_db_prices: defaultdict[Asset, defaultdict[Timestamp, FVal]] = defaultdict(lambda: defaultdict(lambda: ZERO))  # noqa: E501
+        for idx, data in enumerate(query_data):
```

Comment:
```suggestion
        for idx, (asset, _, timestamp) in enumerate(query_data):
```
Using `data[x]` is quite confusing.